UPGRADE_6.0.md

```diff
@@ -0,0 +1,87 @@
+UPGRADE FROM 5.x to 6.0
+=======================
+
+## Introdcure ES6
+
+### Changes in Frontend
+
+#### Social media plugin
+
+In the past there was a ‘shareMenu’ function, which you could use to render share buttons for different social media platforms. We removed this functionality, as it was not maintained anymore. More importantly it was mostly against the terms of different social media platforms. Therefore we recommend you to use the official share-widgets.
+
+### Changes in backend
```

Comment:
Maybe add a section about custom modules for Fork CMS, they need to upgrade the JS files to be `index.js` instead to still get loaded automatically in the module page? Instead of having the name of the module, e.g. Instagram module would auto load `Instagram.js`